File: Leetcode/Median of Two Sorted Arrays.py
class Solution(object):
    def findMedianSortedArrays(self, nums1: List[int], nums2: List[int]) -> float:
        list3 = nums1 + nums2
        list3.sort()
        if len(list3) % 2 != 0:
            l = int(len(list3)/2)
            return list3[l]
        else:
            l = int(len(list3)/2)
            return (list3[l]+list3[l-1])/2

obj1 = Solution()
print(obj1.findMedianSortedArrays([2],[]))

File: Leetcode/Median of Two Sorted Arrays.java
public class MedianTwoSortedArray {
    public double findMedianSortedArrays(int[] nums1, int[] nums2) {
        Vector<Integer> list=new Vector<Integer>();
        int i=0,j=0;
        while(i<nums1.length || j<nums2.length){
            if(i<nums1.length && j<nums2.length){
                if(nums1[i]<nums2[j]){
                    list.add(nums1[i]);i++;
                }else{list.add(nums2[j]);j++;}
            }
            else if(i>=nums1.length){
                list.add(nums2[j]);j++;
            }else{
                list.add(nums1[i]);i++;
            }
        }
        if(list.size()%2 == 0){
            return (double)((list.get(list.size()/2) + list.get(list.size()/2-1)))/2;
        }else{
            return (double)(list.get(list.size()/2));
        }
    }
    public static void main(String args[]){
        MedianTwoSortedArray medianTwoSortedArray = new MedianTwoSortedArray();
        System.out.println(medianTwoSortedArray.findMedianSortedArrays(new int[]{2},new int[]{}));
    }
}
